recipes/datacube/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "datacube" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/opendatacube/datacube-core.git
+  git_tag: datacube-{{ version }}
```

Comment:
We prefer to use zip or tarballs downloads (which should be present for every tagged version on github) because they have a much lower overhead than doing a full git clone. That will mean also adding a checksum (sha256 or md5).